### PR TITLE
Some items and blocks mappings

### PR DESCRIPTION
**Pull request by arthurbambou**

### CODE REVIEW

**Review by hYdos — APPROVED** [review]

File: mappings/net/minecraft/item/Item.mapping [hunk 1]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13224 SNOWBALL Lnet/minecraft/class_2054;
+	FIELD field_13225 BOAT Lnet/minecraft/class_2054;
+	FIELD field_13226 LEATHER Lnet/minecraft/class_2054;
+	FIELD field_13227 MILK Lnet/minecraft/class_2054;
+	FIELD field_13228 BRICK Lnet/minecraft/class_2054;
+	FIELD field_13229 CLAY Lnet/minecraft/class_2054;
+	FIELD field_13230 REEDS Lnet/minecraft/class_2054;
+	FIELD field_13231 PAPER Lnet/minecraft/class_2054;
+	FIELD field_13232 BOOK Lnet/minecraft/class_2054;
+	FIELD field_13233 SLIMEBALL Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:46]: SLIME_BALL

File: mappings/net/minecraft/item/Item.mapping [hunk 2]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13234 MINECART_CHEST Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:47]: CHEST_MINECART

File: mappings/net/minecraft/item/Item.mapping [hunk 3]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13235 MINECART_FURNACE Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:48]: FURNACE_MINECART

File: mappings/net/minecraft/item/Item.mapping [hunk 4]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13254 DIAMOND_HELMET Lnet/minecraft/class_2012;
+	FIELD field_13255 DIAMOND_CHESTPLATE Lnet/minecraft/class_2012;
+	FIELD field_13256 DIAMOND_LEGGINGS Lnet/minecraft/class_2012;
+	FIELD field_13257 DIAMOND_BOOTS Lnet/minecraft/class_2012;
+	FIELD field_13258 GOLD_HELMET Lnet/minecraft/class_2012;
+	FIELD field_13259 GOLD_CHESTPLATE Lnet/minecraft/class_2012;
+	FIELD field_13260 GOLD_LEGGINGS Lnet/minecraft/class_2012;
+	FIELD field_13261 GOLD_BOOTS Lnet/minecraft/class_2012;
+	FIELD field_13262 FLINT Lnet/minecraft/class_2054;
+	FIELD field_13263 PORKCHOP_RAW Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:76]: PORKCHOP

File: mappings/net/minecraft/item/Item.mapping [hunk 5]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13264 PORKCHOP_COOKED Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:77]: COOKED_PORKCHOP

File: mappings/net/minecraft/item/Item.mapping [hunk 6]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13265 PAINTING Lnet/minecraft/class_2054;
+	FIELD field_13266 GOLDEN_APPLE Lnet/minecraft/class_2054;
+	FIELD field_13267 SIGN Lnet/minecraft/class_2054;
+	FIELD field_13268 WOOD_DOOR Lnet/minecraft/class_2054;
+	FIELD field_13269 EMPTY_BUCKET Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:82]: BUCKET

File: mappings/net/minecraft/item/Item.mapping [hunk 7]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13270 WATER_BUCKET Lnet/minecraft/class_2054;
+	FIELD field_13271 BREWING_STAND Lnet/minecraft/class_2054;
+	FIELD field_13272 CAULDRON Lnet/minecraft/class_2054;
+	FIELD field_13273 ENDER_EYE Lnet/minecraft/class_2054;
+	FIELD field_13274 SPECKLED_MELON Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:87]: GLISTERING_MELON_SLICE

File: mappings/net/minecraft/item/Item.mapping [hunk 8]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13275 SPAWN_EGG Lnet/minecraft/class_2054;
+	FIELD field_13276 XP_BOTTLE Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:89]: EXPERIENCE_BOTTLE

File: mappings/net/minecraft/item/Item.mapping [hunk 9]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13277 FIREBALL Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:90]: FIRE_CHARGE

File: mappings/net/minecraft/item/Item.mapping [hunk 10]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13278 WRITABLE_BOOK Lnet/minecraft/class_2054;
+	FIELD field_13279 WRITTEN_BOOK Lnet/minecraft/class_2054;
+	FIELD field_13280 EMERALD Lnet/minecraft/class_2054;
+	FIELD field_13281 ITEM_FRAME Lnet/minecraft/class_2054;
+	FIELD field_13282 FLOWER_POT Lnet/minecraft/class_2054;
+	FIELD field_13283 CARROTS Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:96]: CARROT

File: mappings/net/minecraft/item/Item.mapping [hunk 11]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13284 POTATOES Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:97]: POTATO

File: mappings/net/minecraft/item/Item.mapping [hunk 12]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13285 BAKED_POTATO Lnet/minecraft/class_2054;
+	FIELD field_13286 POISONOUS_POTATO Lnet/minecraft/class_2054;
+	FIELD field_13287 EMPTY_MAP Lnet/minecraft/class_2038;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:100]: MAP

File: mappings/net/minecraft/item/Item.mapping [hunk 13]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13288 GOLDEN_CARROT Lnet/minecraft/class_2054;
+	FIELD field_13289 SKULL Lnet/minecraft/class_2054;
+	FIELD field_13290 CARROT_ON_A_STICK Lnet/minecraft/class_2054;
+	FIELD field_13291 NETHER_STAR Lnet/minecraft/class_2054;
+	FIELD field_13292 PUMPKIN_PIE Lnet/minecraft/class_2054;
+	FIELD field_13293 FIREWORK Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:106]: FIREWORK_ROCKET

File: mappings/net/minecraft/item/Item.mapping [hunk 14]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13294 FIREWORKS_CHARGE Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:107]: FIREWORK_STAR

File: mappings/net/minecraft/item/Item.mapping [hunk 15]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13295 ENCHANTED_BOOK Lnet/minecraft/class_2039;
+	FIELD field_13296 COMPARATOR Lnet/minecraft/class_2054;
+	FIELD field_13297 SUGAR Lnet/minecraft/class_2054;
+	FIELD field_13298 CAKE Lnet/minecraft/class_2054;
+	FIELD field_13299 BED Lnet/minecraft/class_2054;
+	FIELD field_13300 REPEATER Lnet/minecraft/class_2054;
+	FIELD field_13301 COOKIE Lnet/minecraft/class_2054;
+	FIELD field_13302 MAP_FILLED Lnet/minecraft/class_4;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:115]: FILLED_MAP

File: mappings/net/minecraft/item/Item.mapping [hunk 16]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13303 SHEARS Lnet/minecraft/class_18;
+	FIELD field_13304 MELON Lnet/minecraft/class_2054;
+	FIELD field_13305 PUMPKIN_SEEDS Lnet/minecraft/class_2054;
+	FIELD field_13306 MELON_SEEDS Lnet/minecraft/class_2054;
+	FIELD field_13307 RAW_BEEF Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:120]: BEEF

File: mappings/net/minecraft/item/Item.mapping [hunk 17]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13308 COOKED_BEEF Lnet/minecraft/class_2054;
+	FIELD field_13309 RAW_CHICKEN Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:122]: CHICKEN

File: mappings/net/minecraft/item/Item.mapping [hunk 18]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13310 COOKED_CHICKEN Lnet/minecraft/class_2054;
+	FIELD field_13311 ROTTEN_FLESH Lnet/minecraft/class_2054;
+	FIELD field_13312 ENDER_PEARL Lnet/minecraft/class_2054;
+	FIELD field_13313 BLAZE_ROD Lnet/minecraft/class_2054;
+	FIELD field_13314 GHAST_TEAR Lnet/minecraft/class_2054;
+	FIELD field_13315 GOLD_NUGGET Lnet/minecraft/class_2054;
+	FIELD field_13316 NETHER_WART Lnet/minecraft/class_2054;
+	FIELD field_13317 POTION Lnet/minecraft/class_11;
+	FIELD field_13318 EMPTY_BOTTLE Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:131]: GLASS_BOTTLE ?

File: mappings/net/minecraft/item/Item.mapping [hunk 19]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13319 SPIDER_EYE Lnet/minecraft/class_2054;
+	FIELD field_13320 FERMENTED_SPIDER_EYE Lnet/minecraft/class_2054;
+	FIELD field_13321 BLAZE_POWDER Lnet/minecraft/class_2054;
+	FIELD field_13322 MAGMA_CREAM Lnet/minecraft/class_2054;
+	FIELD field_13323 NETHER_BRICK Lnet/minecraft/class_2054;
+	FIELD field_13324 QUARTZ Lnet/minecraft/class_2054;
+	FIELD field_13325 MINECART_TNT Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:138]: TNT_MINECART

File: mappings/net/minecraft/item/Item.mapping [hunk 20]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13326 MINECART_HOPPER Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:139]: HOPPER_MINECART

File: mappings/net/minecraft/item/Item.mapping [hunk 21]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13334 RECORD_BLOCKS Lnet/minecraft/class_2054;
+	FIELD field_13335 RECORD_CHIRP Lnet/minecraft/class_2054;
+	FIELD field_13336 RECORD_FAR Lnet/minecraft/class_2054;
+	FIELD field_13337 RECORD_MALL Lnet/minecraft/class_2054;
+	FIELD field_13338 RECORD_MELLOHI Lnet/minecraft/class_2054;
+	FIELD field_13339 RECORD_STAL Lnet/minecraft/class_2054;
+	FIELD field_13340 RECORD_STRAD Lnet/minecraft/class_2054;
+	FIELD field_13341 RECORD_WARD Lnet/minecraft/class_2054;
+	FIELD field_13342 RECORD_11 Lnet/minecraft/class_2054;
+	FIELD field_13343 RECORD_WAIT Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:156]: RECORD -> MUSIC_DISC

File: mappings/net/minecraft/item/Item.mapping [hunk 22]
```diff
@@ -3,6 +3,178 @@ CLASS net/minecraft/class_2054 net/minecraft/item/Item
...
+	FIELD field_13352 ARROW Lnet/minecraft/class_2054;
+	FIELD field_13353 COAL Lnet/minecraft/class_2054;
+	FIELD field_13354 DIAMOND Lnet/minecraft/class_2054;
+	FIELD field_13355 IRON_INGOT Lnet/minecraft/class_2054;
+	FIELD field_13356 GOLD_INGOT Lnet/minecraft/class_2054;
+	FIELD field_13357 IRON_SWORD Lnet/minecraft/class_2054;
+	FIELD field_13358 WOOD_SWORD Lnet/minecraft/class_2054;
+	FIELD field_13359 WOOD_SHOVEL Lnet/minecraft/class_2054;
+	FIELD field_13360 WOOD_PICKAXE Lnet/minecraft/class_2054;
+	FIELD field_13361 WOOD_AXE Lnet/minecraft/class_2054;
```
Comment (DeadlyMC) [inline mappings/net/minecraft/item/Item.mapping:174]: WOOD -> WOODEN

**Review by DeadlyMC — CHANGES_REQUESTED** [review]
I highly suggest you look at upstream mappings if you make prs anytime again

File: mappings/net/minecraft/block/Block.mapping [hunk 23]
```diff
@@ -2,6 +2,161 @@ CLASS net/minecraft/class_160 net/minecraft/block/Block
...
+	FIELD field_12057 COMPARATOR_ON Lnet/minecraft/class_2106;
+	FIELD field_12058 DAYLIGHT_DETECTOR Lnet/minecraft/class_184;
+	FIELD field_12059 REDSTONE_BLOCK Lnet/minecraft/class_160;
+	FIELD field_12060 QUARTZ_ORE Lnet/minecraft/class_160;
+	FIELD field_12061 HOPPER Lnet/minecraft/class_2130;
+	FIELD field_12062 QUARTZ_BLOCK Lnet/minecraft/class_160;
+	FIELD field_12063 QUARTZ_STAIRS Lnet/minecraft/class_160;
+	FIELD field_12064 ACTIVATOR_RAIL Lnet/minecraft/class_160;
+	FIELD field_12065 DROPPER Lnet/minecraft/class_160;
+	FIELD field_12066 GRAVEL_SOUND Lnet/minecraft/class_162;
```
[the remark this answers is not in the document]
Comment (arthurbambou) [reply mappings/net/minecraft/block/Block.mapping:167]: Yes, block sounds seems to be registered in the same class as blocks

File: mappings/net/minecraft/block/Block.mapping [hunk 24]
```diff
@@ -2,6 +2,161 @@ CLASS net/minecraft/class_160 net/minecraft/block/Block
...
+	FIELD field_12074 BLOCKS [Lnet/minecraft/class_160;
```
[the remark this answers is not in the document]
Comment (arthurbambou) [reply mappings/net/minecraft/block/Block.mapping:168]: There is no registry class for blocks or items, should I call this one registry? It's a Block array

File: mappings/net/minecraft/block/Block.mapping (hunk 23 above)
[the remark this answers is not in the document]
Comment (DeadlyMC) [reply mappings/net/minecraft/block/Block.mapping:167]: oh wow, interesting. Then its fine

File: mappings/net/minecraft/block/Block.mapping (hunk 24 above)
[the remark this answers is not in the document]
Comment (DeadlyMC) [reply mappings/net/minecraft/block/Block.mapping:168]: Leave it as `BLOCKS`

File: mappings/net/minecraft/item/Item.mapping (hunk 1 above)
Comment (DeadlyMC) [reply mappings/net/minecraft/item/Item.mapping:46]: You forgot this one

**Review by DeadlyMC — CHANGES_REQUESTED** [review]
You forgot to remap one of them, otherwise all looks good.

**Review by DeadlyMC — APPROVED** [review]
Looks good! Thanks for the pr. Im gonna hold it until i change the buildscript to publish to our new maven instead of bintray

**Comment by arthurbambou** [comment]
bump

**Comment by DeadlyMC** [comment]
Thank you for the pr, merging now!

**Pull request merged**